ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,56 @@
+|  POLL_INTERVAL | 10 | The interval to wait between each check for Kubernetes resources status change |
+|  KUBERNETES_VERSION | latest_stable | The Kubernetes version (or branch) to build |
+|  HARBOR_REGESTRY | harbor.mellanox.com | The docker registry to pull images from |
+|  HARBOR_PROJECT | cloud-orchestration | The docker registry project to pull the images from |
+|  MULTUS_CNI_REPO | <https://github.com/intel/multus-cni> | Multus CNI repository URL |
+|  MULTUS_CNI_BRANCH | master | Multus CNI branch to build |
+|  MULTUS_CNI_PR || Multus CNI pull request number to pull, if this is used the MULTUS_CNI_BRANCH is ignored |
+|  MULTUS_CNI_HARBOR_IMAGE | ${HARBOR_REGESTRY}/${HARBOR_PROJECT}/multus | The Multus image to pull in case the project was not built |
+|  PLUGINS_REPO | <https://github.com/containernetworking/plugins.git> | containernetworking repository URL |
+|  PLUGINS_BRANCH | master | containernetworking branch to build |
+|  PLUGINS_BRANCH_PR || containernetworking pull request number to pull, if this is used the PLUGINS_BRANCH is ignored |
+|  SRIOV_NETWORK_DEVICE_PLUGIN_REPO | <https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin> | SRIOV network device plugin repository to use |
+|  SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH | master | SRIOV network device plugin branch to build |
+|  SRIOV_NETWORK_DEVICE_PLUGIN_PR || SRIOV network device plugin pull request number to pull, adding this will ignore SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH |
+|  SRIOV_NETWORK_DEVICE_PLUGIN_HARBOR_IMAGE | ${HARBOR_REGESTRY}/${HARBOR_PROJECT}/sriov-device-plugin | The SRIOV network device plugin image to pull in case the project was not built |
+|  GOPATH | ${WORKSPACE} ||
+|  PATH | /usr/local/go/bin/:$GOPATH/src/k8s.io/kubernetes/third_party/etcd:$PATH ||
+|  CNI_BIN_DIR | /opt/cni/bin/ | This is used to configure Kubernetes local_cluser_up.sh CNI_BIN_DIR |
+|  CNI_CONF_DIR | /etc/cni/net.d/ | This is used to configure Kubernetes local_cluser_up.sh CNI_CONF_DIR |
+|  API_HOST | $(hostname) | The Node name to use |
+|  API_HOST_IP | $(hostname -I | awk '{print $1}') | The API server IP |
+|  POD_CIDR | 192.168.0.0/16 | The Pods network subnet |
+|  SERVICE_CIDR | 172.0.0.0/16 | The Service network subnet |
+|  KUBECONFIG | /etc/kubernetes/admin.conf | The KUBECONFIG file to use |
+|  SRIOV_INTERFACE | auto_detect | The Mellanox interface to use to create the VFs |
+|  NETWORK | "192.168.$N" | This is used to setup the MACVLAN network range, N is randomly generated |
+|  ANTREA_CNI_REPO | <https://github.com/vmware-tanzu/antrea.git> | Antrea project repository to use |
+|  ANTREA_CNI_BRANCH | master | Antrea project branch to use |
+|  ANTREA_CNI_PR | | Antrea project pull request number to pull, adding this will ignore ANTREA_CNI_BRANCH |
+|  ANTREA_CNI_HARBOR_IMAGE | ${HARBOR_REGESTRY}/${HARBOR_PROJECT}/antrea | The Antrea image to pull in case the project was not built |
```

Comment:
just want to confirm that `ANTREA_CNI_HARBOR_IMAGE` is not used when testing a pull request, since then the image would be built?

Reply:
That is correct, the images is not used in case of pull requests. The image is built nightly from the Antrea master branch and used in a daily CI. For pull requests the ci will try to build the image from the fetched pull request, and uses the built image. see <https://github.com/vmware-tanzu/antrea/pull/1729/files?file-filters%5B%5D=.sh#diff-12b01860bd03aec02f37d67c6cec364f88591242e1ad900fe68e915ba3a41ec1R73-R87>